source/voice_control/android.markdown

```diff
@@ -65,11 +65,16 @@ The way how Assist can be set up on your phone may differ depending on your vers
 
 2. After selecting **Assist**, wait for **How can I assist?** to be displayed and the microphone to start pulsing.
    
-    ![Assist button](/images/assist/android_watch_2.png)
+    ![How can I assist](/images/assist/android_watch_2.png)
 
 3. Speak your command.
    
-    ![Assist button](/images/assist/android_watch_3.png)
+    ![Assist speak your command](/images/assist/android_watch_3.png)
+
+4. To change the assistant, select the current assistant (**Home Assistant Cloud** in the screenshot above).
```

Comment:
Maybe "tap" instead of "select"?

Reply:
Excellent point. Thank you for the terminology catch.